lab-lindsay/test/server-test.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const server = require('../server');
+const cowsay = require('cowsay');
+const chai = require('chai');
+const http = require('chai-http');
+const expect = chai.expect;
+
+chai.use(http);
+
+describe('Server module', function() {
+  before(done => {
+    server.listen(3000);
+    done();
+  });
+
+  describe('POST method', function() {
+    describe('/ endpoint', function() {
+      it('should respond with a 400 on a bad request', done => {
+        chai.request(server);
```

Comment:
**TODO FOR LOST POINTS**

Your test breaks because you need to remove your semi-colon at Line 20.